foia_hub/templates/learn.html

```diff
@@ -25,6 +25,8 @@
         </ul>
       </nav>
 
+        <h2>Didn't find the information you're looking for? Check <a href="FOIA.gov/about.html">FOIA.gov/Learn</a></h2>
```

Comment:
`<a href="http://foia.gov/about.html">` or replace http with https if that's your thing.